app/helpers/admin_helper.rb

```diff
@@ -50,11 +50,11 @@ def admin_menu_items
   # @ridhwana end delete
 
   def deduced_controller(request)
-    request.path.split("/").last
+    request.path.split("/").fourth
```

Comment:
Even though this seems a bit hacky it makes more sense to get the deduced controller and scope from left to right to account for nested routes